src/index.ts

```diff
@@ -467,10 +519,11 @@ export class Bigtable {
     const dataOptions = Object.assign(
       {},
       baseOptions,
+      getUniverseDomainOptions(options, options.BigtableClient),
       {
         servicePath:
           customEndpointBaseUrl ||
-          getDomain('bigtable', options.BigtableClient),
+          getDomain('bigtable', options, options.BigtableClient),
```

Comment:
This now supports the use case where the universe domain is provided in the handwritten client options as well as maintaining support for when the universe domain is provided just for the gapic client.

Reply:
Then it would be set to `bigtable.${universeDomain}` where universeDomain is the value provided in the Gapic client.